extensions/typescript/src/features/referenceProvider.ts

```diff
@@ -14,7 +14,7 @@ export default class TypeScriptReferenceSupport implements ReferenceProvider {
 	public constructor(
 		private client: ITypescriptServiceClient) { }
 
-	public provideReferences(document: TextDocument, position: Position, options: { includeDeclaration: boolean }, token: CancellationToken): Promise<Location[]> {
+	public provideReferences(document: TextDocument, position: Position, options: { includeDeclaration: boolean }, _: (value: Location) => void, token: CancellationToken): Promise<Location[]> {
```

Comment:
Why are you not passing the token first and then the progress callback as an optional paramter with default value of a noop?